pydantic_ai_slim/pydantic_ai/messages.py

```diff
@@ -465,6 +484,13 @@ class ModelResponse:
     kind: Literal['response'] = 'response'
     """Message type identifier, this is available on all parts as a discriminator."""
 
+    # TODO: implemented in OpenAI, but not in other models
+    logprobs: list[ChatCompletionTokenLogprob] = field(default_factory=lambda: [])
```

Comment:
```suggestion
    logprobs: list[ChatCompletionTokenLogprob] | None = field(None, repr=False)
```
Let's exclude from the repr by default. I also feel like it's worth using `None` to indicate whether it was included in the response. I guess that will make the ergonomics a bit worse; I'd be okay always including it if it doesn't have performance consequences for non-users and will be empty if and only if it wasn't returned.